doc/sphinx/changes.rst

```diff
@@ -8,6 +8,991 @@ Recent changes
 
    .. include:: ../unreleased.rst
 
+Version 8.20
+------------
+
+Summary of changes
+~~~~~~~~~~~~~~~~~~
+
+Coq version 8.20 adds a new rewrite rule mechanism along with a few
+new features, a host of improvements to the virtual machine, the
+notation system, Ltac2 and the standard library.
+
+We highlight some of the most impactful changes here:
+
+  - :ref:`rewrite_rules`
+
+  - :ref:`primitive strings <pstring_library>`
+
+  - A lot of work went into reducing the size of the bytecode segment,
+    which in turn means that .vo files might now be considerably
+    smaller.
+
+  - A new version of the Docker keeper tool to build and maintain
+    Docker images of Coq.
```

Comment:
@Zimmi48 @erikmd could you please check that sentence and provide a link to point to?